### PR TITLE
feat(trigger): organization level audit configs

### PR DESCRIPTION
**Pull request by ekremney**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/controllers/trigger/common/trigger.js [hunk 1]
```diff
@@ -14,6 +14,36 @@ import { internalServerError, notFound, ok } from '@adobe/spacecat-shared-http-u
...
+
+  return orgAuditConfig.auditsDisabled() || siteAuditConfig.auditsDisabled();
+}
+
+function isAuditTypeDisabled(auditType, site, organization) {
+  const orgAuditDisabled = organization.getAuditConfig().getAuditTypeConfig(auditType)?.disabled();
+  const siteAuditDisabled = site.getAuditConfig().getAuditTypeConfig(auditType)?.disabled();
+
+  return orgAuditDisabled || siteAuditDisabled;
+}
```
Comment (solaris007) [inline src/controllers/trigger/common/trigger.js:29]: This logic will probably be heavily re-used in audit-worker and others, should we centralize?

**Review by solaris007 — APPROVED** [review]

File: src/controllers/trigger/common/trigger.js (hunk 1 above)
Comment (ekremney) [reply src/controllers/trigger/common/trigger.js:29]: please see: https://github.com/adobe/spacecat-shared/pull/173

**Comment by iuliag** [comment]
@ekremney, could you please also update the API spec with the changes introduced here?
E.g.:
https://github.com/adobe/spacecat-api-service/blob/main/docs/openapi/schemas.yaml#L82 doesn't mention audits, nor the example of the entity https://github.com/adobe/spacecat-api-service/blob/main/docs/openapi/schemas.yaml#L119 or of the create/update request body https://github.com/adobe/spacecat-api-service/blob/main/docs/openapi/schemas.yaml#L160

**Pull request merged**